worlds/sc2wol/Items.py

```diff
@@ -184,7 +184,7 @@ def get_basic_units(multiworld: MultiWorld, player: int) -> typing.Set[str]:
 
 defense_ratings = {
     "Siege Tank": 5,
-    "Maelstrom Rounds": 2,
+    # "Maelstrom Rounds": 2,
```

Comment:
is this an intentional change? If so, maybe a comment explaining it would be nice.